packages/core/src/Core.ts

```diff
@@ -58,6 +58,7 @@ export class Core<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
             const Separator = this.editor.configuration.defaults.Separator;
             range.start.before(new Separator());
         }
+        range.updateModifiers();
```

Comment:
My gut feeling is this: if we have to manually invalidate the cache more often than we used to when it was in Inline then we are doing something wrong.

I think something went wrong here as the original idea was to simply change the place it was stored at.

I'm not reviewing further down until we sort this out. (Except the other comment I already wrote before this one.)

Reply:
Well we could invalidate the cash the same way as we did before: on `insertText`. But I believe that was wrong to begin with.